test/EmitVerilog/verilog-basic.fir

```diff
@@ -183,11 +182,11 @@ circuit inputs_only :
 ; CHECK-NEXT:     assign out = $signed(c) >= $signed(d);
 ; CHECK-NEXT:     assign out = $signed(a) >= $signed(b);
 ; CHECK-NEXT:     assign out = a == b;
-; CHECK-NEXT:     assign out = c == d;
-; CHECK-NEXT:     assign out = a == b;
+; CHECK-NEXT:     assign out = $signed(c) == $signed(d);
+; CHECK-NEXT:     assign out = $signed(a) == $signed(b);
```

Comment:
:-(

Reply:
yea, assuming firrtl sign always matters causes this.  It could be eliminated if one inverts the old strictSign flag to be a strinctUnsigned flag...

Reply:
Ok, I don't care about quality of result for the firrtl dialect, since we're on a path to eliminating it.  I do care about this sort of thing for the RTL dialect though.  We probably have a bunch of work to get there though.